### DISCUSSION TITLE
关于chatsydney的网页端部署……

### DISCUSSION THREAD

**Discussion by Abcdefghuuj** [viopsa233/AI-testers-Forum#76] [Q&A] [answered]
刷阿B的时候发现了这个仓
https://github.com/adams549659584/go-proxy-bingai
又想起@SoraRoseous搭建的nekointelligence.cyou:65432，国内搭梯子又不好用。
把Sora用的chatsydney像adams这样部署有可行的方法吗？
只要能像bing.vcanbb.top这样好的访问性就行了🤔

**Comment by SoraRoseous** [comment]
![image](https://github.com/viopsa233/AI-testers-Forum/assets/54582495/7f107227-2050-48df-aef2-dd0405d21f0c)
这个ip是国外的吧🤔

**Reply by SoraRoseous** [reply, marked as answer]
https://bing.xuhai.ink/
这个网站的服务器就是海外的，另外一个大佬搭的

**Comment by yinghua8wu** [comment]
https://huggingface.co/spaces/XM4HFl2RbY/RDDdYNgD8D
复制空间就行，个人用的话，可以改成私人空间，然后在Dockerfile里加一行
ENV Go_Proxy_BingAI_USER_TOKEN=xxxxxxxxx（Cookie值）
